### ISSUE TITLE
a11y testing results Iphone SE

### ISSUE THREAD

**Issue by philli-m** [liqd/zt-app#343]
**Login**
- [x] text inputs don't appear to be associated with labels
- [ ] links not read out as links just normal text fixed for solo link but inline links won't work, consider design change
- [ ] helptext not read out like in web forms when form errors

**Explore**
- [ ] project tile not read as link

**Project**
- [ ] idea tile not read as link
- [ ] vote and comment numbers have no context, only numbers read
- [ ] labels not identified as labels

**Idea Form**
- [ ] first field read is 'add image'
- [ ] idea description not read as link
- [ ] Idea category dropdown not read as select input
- [ ] tabbing (next btn) doesn't lead to next form field
- [ ] description textarea submit btn hidden
- [ ] close photo btn no context 
- [ ] can't click img checkbox

**Comment by phillimorland** [comment]
results of automated a11y linting
```
/home/phillippa/zt-app/components/ButtonCounter.js
  24:5  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/components/ButtonTextInput.js
  22:5  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/components/RichtextCollapsibleItem.js
  14:5  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/components/imageFormField.js
  113:13  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop  react-native-a11y/has-valid-accessibility-ignores-invert-colors
  139:13  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop  react-native-a11y/has-valid-accessibility-ignores-invert-colors

/home/phillippa/zt-app/containers/Comments/Comment.js
  198:11  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop                                                                      react-native-a11y/has-valid-accessibility-ignores-invert-colors
  233:24  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
  238:49  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/containers/Comments/CommentForm.js
  43:15  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
  56:15  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/containers/Comments/SubComment.js
  34:11  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop                                                                      react-native-a11y/has-valid-accessibility-ignores-invert-colors
  68:24  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors

/home/phillippa/zt-app/containers/Ideas/ExploreListItem.js
  17:5   error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
  20:11  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop                                                                      react-native-a11y/has-valid-accessibility-ignores-invert-colors

/home/phillippa/zt-app/containers/Ideas/Idea.js
  329:9   error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
  338:17  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop                                                                      react-native-a11y/has-valid-accessibility-ignores-invert-colors

/home/phillippa/zt-app/containers/Ideas/IdeaCreateDescription.js
  12:10  error  'VirtualScrollView' is defined but never used. Allowed unused vars must match /(React)/u  no-unused-vars

/home/phillippa/zt-app/containers/Ideas/IdeaProject.js
  115:19  error  Found an element which will be inverted. Add the accessibilityIgnoresInvertColors prop  react-native-a11y/has-valid-accessibility-ignores-invert-colors

/home/phillippa/zt-app/containers/Ideas/IdeasListItem.js
  39:5  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
```
